linux/datadog_wrapper

```diff
@@ -1,96 +1,165 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
+
+main() {
+    # Required to add the AAS metadata
+    export DD_AZURE_APP_SERVICES=1
+
+    echo "Setting application directory as $APP_DIR"
+    APP_DIR=$(pwd)
+
+    echo "Setting Datadog environment variables"
+    setEnvVars
+
+    echo "Creating and switching to the Datadog directory at $DD_DIR"
+    mkdir -p "$DD_DIR" && cd "$DD_DIR" || return
+
+    echo "Getting the required Datadog binaries"
+    STARTUP_PIDS=()
+    getBinaries
+
+    case "$WEBSITE_STACK" in
+        "NODE")
+            echo "Initializing setup for a Node environment"
+            setUpNodeEnv;;
+        "DOTNETCORE")
+            echo "Initializing setup for a .NET environment"
+            setUpDotnetEnv;;
+        "*")
+            echo "Unsupported runtime. Exiting Datadog startup"
```

Comment:
Interestingly, you've already started the trace agent and dogstatsd by this point. I guess that's fine? This is actually a regression though because previously, we would skip starting the binaries if the language is unsupported.